src/refract/RenderJSONVisitor.cc

```diff
@@ -102,10 +105,38 @@ namespace refract
         if (!a || a->value.empty()) {
             return NULL;
         }
+        
+        std::cerr << "samples" << std::endl;
 
         return TypeQueryVisitor::as<T>(*(a->value.begin()));
     }
 
+    template<typename T>
+    const T* getNullable(const T& e) {
+        IElement::MemberElementCollection::const_iterator ta = e.attributes.find("typeAttributes");
+        
+        if (ta == e.attributes.end()) {
+            return NULL;
+        }
+        
+        SerializeCompactVisitor s;
+        s.visit(*(*ta));
+        
+        std::cerr << s.key() << std::endl;
+        
+        if(s.value().type == 5)
+        {
+            std::cerr << s.key() << std::endl;
+            
+            for(int x = 0; x <= s.value().array().size(); x++)
+            {
+                std::cerr << s.value().array()[0].str << std::endl;
+            }
+        }
+        
+        return NULL;
```

Comment:
@pksunkara what do you suggest we return when we find `nullable`, NULL almost makes sense but obviously could be confusing.